Unable to install because https://repo.dev.wixpress.com is continually unreachable

Packages in package-lock.json require downloads from Wix's npm repos but running `npm install` gives timeout errors for any package served from the url in the title of this issue.

```
$ npm install
npm ERR! code ENETUNREACH
npm ERR! syscall connect
npm ERR! errno ENETUNREACH
npm ERR! request to https://repo.dev.wixpress.com/artifactory/api/npm/npm-repos/touch/-/touch-3.1.0.tgz failed, reason: connect ENETUNREACH 10.42.121.120:443
```

--- a/README.md
+++ b/README.md
@@ -84,7 +84,6 @@ E. Run your app
 
 * Browse to the cloned sample application
 * Run `npm install`
-* Run `npm build`
 * Run `npm start`
 
 You should get something like this:

